tools/enso4igv/package-lock.json

```diff
@@ -1722,10 +1722,11 @@
       "dev": true
     },
     "node_modules/js-yaml": {
-      "version": "3.14.1",
```

Comment:
`js-yaml` is, obviously, a transitive dependency. I don't see such dependency in `package.json`. Is it safe to update just a transitive dependency? Why is dependabot allowed to change only `package-lock.json`?